plugins/net_plugin/include/eosio/net_plugin/auto_bp_peering.hpp

```diff
@@ -132,7 +132,7 @@ class bp_connection_manager {
 
    // Only called from main thread
    void on_pending_schedule(const chain::producer_authority_schedule& schedule) {
-      if (auto_bp_peering_enabled() && self()->in_sync()) {
+      if (auto_bp_peering_enabled() && self()->not_lib_catchup()) {
```

Comment:
Somewhere you can add a comment (like the one in an early commit message) saying in Savana, not in lib_catchup implies almost in sync.

Reply:
Done